dist/index.html

```diff
@@ -28,20 +28,16 @@ <h2 class="user-name-js"></h2>
         <section class="make-booking-section">
           <p class="mini-heading">book a room</p>
           <div class="book-a-room-container">
-            <div>
+            <div class="center">
               <div>
                 <form>
                   <label for="calendarDate">select a date:</label><br/>
-                  <input type="date" id="calendarDate" name="checkin" min="2022-01-18" max="2022-06-01">
+                  <input type="date" id="calendarDate"
+                   name="checkin" min="2022-01-18" max="2022-06-01" value="2022-01-18">
                   <input type="submit" id="selectDateButton" value="search">
                 </form>
               </div>
-              <div>
-                <!-- <p>room type</p>
-                <button>single room</button>
-                <button>junior suite</button>
-                <button>suite</button>
-                <button>residential suite</button> -->
+              <div class="room-type-container-js hidden">
```

Comment:
I was going to do a bunch of error-handling type stuff here to prevent anyone from clicking down on the dropdown until they'd selected a date. Then I realized that I could just hide the dropdown until AFTER they'd selected a date. So much easier.